### PR TITLE
Landing bugs + add username to signup/login forms

### PR DESCRIPTION
**Pull request by dchan3**

### CODE REVIEW

File: imports/api/shows/shows_schema.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default ShowsSchema = new SimpleSchema({
       if (this.isSet || this.isUpdate)
         return this.value;
```
Comment (daveynight) [inline imports/api/shows/shows_schema.js:24]: Are we guaranteed non-null `this.value` when the condition is true? If not, maybe `return this && this.value` would be a little safer.

File: imports/api/shows/shows_schema.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default ShowsSchema = new SimpleSchema({
...
       else
-        return Profiles.find({userId: Meteor.userId}).name;
+        return Profiles.find({ userId: Meteor.userId() }).name;
```
Comment (daveynight) [inline imports/api/shows/shows_schema.js:49]: :))

File: imports/startup/lib/helpers.js [hunk 3]
```diff
@@ -8,11 +8,28 @@ export const currentPlaylist = function() {
...
+             parseInt(this.startTime.split(":")[0]) <= new Date().getHours();
+    }
+  }, { sort: { startTime: -1 } });
+
+  if (playlist && now.getHours() >= parseInt(playlist.endTime.split(":")[0])) {
+    return undefined;
+  }
+  else {
+    return playlist;
+  }
```
Comment (daveynight) [inline imports/startup/lib/helpers.js:32]: Looks like good stuff here 😺

**Review by daveyshindig — APPROVED** [review]

File: imports/api/shows/shows_schema.js (hunk 1 above)
Comment (dchan3) [reply imports/api/shows/shows_schema.js:24]: I think it's implied that `this` is neither undefined nor null when the condition is satisfied, no?

Comment (daveynight) [reply imports/api/shows/shows_schema.js:24]: I wasn't sure, but OK! 👅

**Pull request merged**